src/Components/Filtering/MultiselectFilterOptions.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+
+export default function MultiselectComponent({
+  filterOptions, input, setInput, labelName,
+}) {
+  const result = filterOptions ? filterOptions.split(',') : null;
+  const handleToggle = (val, label) => {
+    setInput({ ...input, [label]: val });
+  };
+
+  // console.log(input);
+  return (
+    filterOptions ? (
+      <Autocomplete
+        multiple
+        options={result}
+        filterSelectedOptions
+        onChange={(event, value) => handleToggle(
+          value,
+          labelName,
+          event,
+        )}
+        renderInput={(params) => (
+          <TextField
+            {...params}
+            label={labelName}
+            placeholder={filterOptions ? `'${filterOptions[0]}...'` : null}
```

Comment:
for some reason the placeholder only shows the first letter rather than the first word